source/draw/Image.ooc

```diff
@@ -27,6 +27,11 @@ CoordinateSystem: enum {
 	YUpward = 0x02
 }
 
+TransformMethod: enum {
```

Comment:
1. I don't like the name `TransformMethod`, it's more a `Filter` or `Interpolation` setting, or if you have a better suggestion. Other than that we should have it in `Image` and adapt the code in `GpuImage`.
2. We need to decide if this should be a state for the Image or a parameter. Currently for GpuImages, filtering is a state for the Image that is being drawn (sampled). The good thing about this is that you don't have to supply it as a parameter and it also handles the case with draw calls where you have multiple images as input (for example in a shader) and only some of them should use bilinear filtering. Also, in this specific case there would be no need for an overloaded `resizeTo`.